homeassistant/components/devialet/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "devialet",
+  "name": "Devialet",
+  "config_flow": true,
```

Comment:
manifest.json needs sorted now.

domain, then name, then the rest of the keys alphabetically.
(ref https://github.com/home-assistant/core/pull/87020)